scripts/androidBeforeBuild.js

```diff
@@ -0,0 +1,25 @@
+module.exports = function(ctx) {
+    var fs = ctx.requireCordovaModule('fs'),
+        path = ctx.requireCordovaModule('path'),
+        gradle = ctx.requireCordovaModule(path.join(ctx.opts.projectRoot, 'platforms/android/cordova/lib/builders/builders')).getBuilder('gradle'),
+        spawn = ctx.requireCordovaModule('cordova-common').superspawn.spawn,
+        deferral = ctx.requireCordovaModule('q').defer();
+
+    var platformRoot = path.join(ctx.opts.projectRoot, 'www');
```

Comment:
@edewit does this mean the `google-services.json` file needs to be in `www` folder of the app ?